projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -448,6 +448,52 @@ private ClientBuilder getClientBuilder() throws Exception {
             .register(MultiPartFeature.class);
    }
 
+   /**
+    * Return true if successfully get the information of the container,
+    * false if container {@code containerName} does not exist or the api
+    * key that is using has no access to the container
+    */
+   public String getContainer(String containerName) {
+      try {
+         Client client = getClientBuilder().build();
+         WebTarget webTarget = getTarget(
+               client,
+               CoordConsts.SVC_RSC_GET_CONTAINER);
+
+         MultiPart multiPart = new MultiPart();
+         multiPart.setMediaType(MediaType.MULTIPART_FORM_DATA_TYPE);
+
+         addTextMultiPart(multiPart, CoordConsts.SVC_KEY_API_KEY,
+               _settings.getApiKey());
+         addTextMultiPart(multiPart, CoordConsts.SVC_KEY_VERSION,
+               Version.getVersion());
+         addTextMultiPart(multiPart, CoordConsts.SVC_KEY_CONTAINER_NAME,
+               containerName);
+
+         Response response = webTarget
+               .request(MediaType.TEXT_PLAIN)
+               .post(Entity.entity(multiPart, multiPart.getMediaType()));
+
+         _logger.debug(response.getStatus() + " " + response.getStatusInfo()
+               + " " + response + "\n");
+
+         if (response.getStatus() != Response.Status.OK.getStatusCode()) {
+            _logger.errorf("GetContainer: Did not get an OK response\n");
+            _logger.errorf(response.readEntity(String.class) + "\n");
+            return null;
+         }
+
+         String testrigs = response.readEntity(String.class) + "\n";
+         return testrigs;
```

Comment:
Consistent with server-side code, use 'containerInfo' as variable name instead of 'testrigs'?